src/target/cortexa.c

```diff
@@ -317,6 +330,54 @@ static size_t create_tdesc_cortex_a(char *buffer, size_t max_len)
 	return (size_t)total;
 }
 
+typedef struct bitfield_entry {
```

Comment:
Given how much extra Flash this'll use, could this please be topped and tailed in `#ifdef ENABLE_DEBUG` (including down to the end of `cortexa_decode_bitfields()`) please?

Reply:
The PR uses just extra 64..128 bytes on different all_platforms binaries, and +6 KiB on hosted. I usually trust the toolchain to drop unused functions and sections. But sure, I can hide the helper before compile-time.